cpp/test/unit/mesh/cmap.ufl

```diff
@@ -0,0 +1,10 @@
+coord_element = VectorElement("Lagrange", triangle, 1)
+mesh = Mesh(coord_element)
+
+element = FiniteElement("Lagrange", triangle, 1)
+V = FunctionSpace(mesh, element)
+
+u = TrialFunction(V)
```

Comment:
Do we need these lines?

Reply:
Not really, but if I leave only the first two lines:
```
coord_element = VectorElement("Lagrange", triangle, 1)
mesh = Mesh(coord_element)
```
I get:
```
line 82, in main
    code_h, code_c = compiler.compile_ufl_objects(
.....
if isinstance(ufl_objects[0], ufl.form.Form):
IndexError: list index out of range
```
Would that be a ffcx issue?

Reply:
Probably no-one generates a code just for cmap. If we'd like to allow this then some minor changes are needed in FFCx, register an issue.

Reply:
Simple/palliative solution on the ffcx side:

https://github.com/FEniCS/ffcx/compare/igor/compile-meshes